modules/InfiniteStories/StoriesList.module.scss

```diff
@@ -7,20 +7,18 @@
 }
 
 .storiesContainer {
+    display: grid;
+    gap: $spacing-6 $grid-gutter-tablet;
+
     @include mobile-only {
-        display: grid;
-        grid-template-columns: repeat(1, 1fr);
-        row-gap: $spacing-6;
+        grid-template-columns: 1fr;
```

Comment:
Sorry, missed that part in previous review:
Let's move this line to the root and use `tablet-up` mixin instead of `tablet-only` to make things more streamlined.